styles/base.js

```diff
@@ -144,8 +141,10 @@ export default () => `
   /* to be used by our code blocks */
   .prism-code {
     display: block;
-    white-space: pre;
-
+    white-space: pre-wrap !important;
```

Comment:
These styles in particular are borrowed from react-live, which injects them again. We could probably tell react-live to not inject the styles. But this is ok for now.